core/src/main/java/edu/wpi/grip/core/events/RunPipelineEvent.java

```diff
@@ -0,0 +1,16 @@
+package edu.wpi.grip.core.events;
+
+
+/**
+ * Any event that indicates to the {@link edu.wpi.grip.core.PipelineRunner} that it should run.
+ */
+public interface RunPipelineEvent {
+
+    /**
+     * Indicates to the {@link edu.wpi.grip.core.PipelineRunner} that there is an update to one of the values and it should run again.
+     * @return true if the pipeline should be released to run because of the update
+     */
+    default boolean pipelineShouldRun() {
```

Comment:
This makes no sense.  Things can send `RunPipelineEvent`s, but sometimes it doesn't mean run?